tools/shared.py

```diff
@@ -1501,7 +1500,6 @@ def nop(arg):
     env['NM'] = quote(LLVM_NM)
     env['LDSHARED'] = quote(unsuffixed(EMCC)) if not WINDOWS else 'python %s' % quote(EMCC)
     env['RANLIB'] = quote(unsuffixed(EMRANLIB)) if not WINDOWS else 'python %s' % quote(EMRANLIB)
-    env['EMMAKEN_COMPILER'] = quote(Building.COMPILER)
```

Comment:
this is mentioned in the docs under `site/`, should we remove it from there too?

Reply:
I'm not removing `EMMAKEN_COMPILER` here.  I'm just not setting it here during autoconf/cmake, since it would just be set to the default anyway.

As a followup change I may try to remove `EMMAKEN_COMPILER`